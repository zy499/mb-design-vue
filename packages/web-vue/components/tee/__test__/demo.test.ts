import demoTest from '../../../scripts/demo-test';

demoTest('tee');
