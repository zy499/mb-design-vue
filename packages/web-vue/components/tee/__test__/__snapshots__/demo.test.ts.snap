// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<tee> demo: render [basic] correctly 1`] = `
"<div class=\\"mb-space mb-space-horizontal mb-space-align-center\\">
  <div class=\\"mb-space-item\\">
    <div class=\\"mb-tee\\">测试组件</div>
  </div>
</div>"
`;
